docs/dev/analysis/cht-waterfall-chart.rst

```diff
@@ -0,0 +1,812 @@
+            <a:schemeClr val="tx1">
+              <a:lumMod val="15000"/>
+              <a:lumOff val="85000"/>
+            </a:schemeClr>
+          </a:solidFill>
+          <a:round/>
+        </a:ln>
+      </cs:spPr>
+      <cs:defRPr sz="1197"/>
+    </cs:seriesAxis>
+    <cs:seriesLine>
+      <cs:lnRef idx="0"/>
+      <cs:fillRef idx="0"/>
+      <cs:effectRef idx="0"/>
+      <cs:fontRef idx="minor">
+        <a:schemeClr val="tx1"/>
+      </cs:fontRef>
+      <cs:spPr>
+        <a:ln w="9525" cap="flat">
+          <a:solidFill>
+            <a:srgbClr val="D9D9D9"/>
+          </a:solidFill>
+          <a:round/>
+        </a:ln>
+      </cs:spPr>
+    </cs:seriesLine>
+    <cs:title>
+      <cs:lnRef idx="0"/>
+      <cs:fillRef idx="0"/>
+      <cs:effectRef idx="0"/>
+      <cs:fontRef idx="minor">
+        <a:schemeClr val="tx1">
+          <a:lumMod val="65000"/>
+          <a:lumOff val="35000"/>
+        </a:schemeClr>
+      </cs:fontRef>
+      <cs:defRPr sz="1862"/>
+    </cs:title>
+    <cs:trendline>
+      <cs:lnRef idx="0">
+        <cs:styleClr val="auto"/>
+      </cs:lnRef>
+      <cs:fillRef idx="0"/>
+      <cs:effectRef idx="0"/>
+      <cs:fontRef idx="minor">
+        <a:schemeClr val="tx1"/>
+      </cs:fontRef>
+      <cs:spPr>
+        <a:ln w="19050" cap="rnd">
+          <a:solidFill>
+            <a:schemeClr val="phClr"/>
+          </a:solidFill>
+          <a:prstDash val="sysDash"/>
+        </a:ln>
+      </cs:spPr>
+    </cs:trendline>
+    <cs:trendlineLabel>
+      <cs:lnRef idx="0"/>
+      <cs:fillRef idx="0"/>
+      <cs:effectRef idx="0"/>
+      <cs:fontRef idx="minor">
+        <a:schemeClr val="tx1">
+          <a:lumMod val="65000"/>
+          <a:lumOff val="35000"/>
+        </a:schemeClr>
+      </cs:fontRef>
+      <cs:defRPr sz="1197"/>
+    </cs:trendlineLabel>
+    <cs:upBar>
+      <cs:lnRef idx="0"/>
+      <cs:fillRef idx="0"/>
+      <cs:effectRef idx="0"/>
+      <cs:fontRef idx="minor">
+        <a:schemeClr val="dk1"/>
+      </cs:fontRef>
+      <cs:spPr>
+        <a:solidFill>
+          <a:schemeClr val="lt1"/>
+        </a:solidFill>
+        <a:ln w="9525">
+          <a:solidFill>
+            <a:schemeClr val="tx1">
+              <a:lumMod val="15000"/>
+              <a:lumOff val="85000"/>
+            </a:schemeClr>
+          </a:solidFill>
+        </a:ln>
+      </cs:spPr>
+    </cs:upBar>
+    <cs:valueAxis>
+      <cs:lnRef idx="0"/>
+      <cs:fillRef idx="0"/>
+      <cs:effectRef idx="0"/>
+      <cs:fontRef idx="minor">
+        <a:schemeClr val="tx1">
+          <a:lumMod val="65000"/>
+          <a:lumOff val="35000"/>
+        </a:schemeClr>
+      </cs:fontRef>
+      <cs:defRPr sz="1197"/>
+    </cs:valueAxis>
+    <cs:wall>
+      <cs:lnRef idx="0"/>
+      <cs:fillRef idx="0"/>
+      <cs:effectRef idx="0"/>
+      <cs:fontRef idx="minor">
+        <a:schemeClr val="tx1"/>
+      </cs:fontRef>
+    </cs:wall>
+  </cs:chartStyle>
```

Comment:
This is too long, let's find a way to summarize, like show a couple styles and then abbreviate the rest.